### ISSUE TITLE
TS type definitions

### ISSUE THREAD

**Issue by pekam** [vaadin/vaadin-messages#42]
Make sure that all the features have type definitions.

Discuss with the components team, if we should use some generator tool or type the definitions fully manually.

**Comment by Peppe** [comment]
one such definition: items in MessageList
```
export default interface MessageListItem {
  text?: string;
  time?: string;
  userName?: string;
  userAbbr?: string;
  userColorIndex?: number;
  userImg?: string;
}

```

**Comment by pekam** [comment]
Resolved by #53
